tools/cmdautotest/README_CMDAutoTest.md

```diff
@@ -3,7 +3,7 @@
 The CMD autotest automatically performs the tasks you need to perform when performing multiple tests on the TASH command and determines the results.
 
 ### Prerequisites
-Install python2.7, pyserial(at pip)
+Install python3, pyserial(at pip3)
```

Comment:
Does pip3 exist..?
As far as I know, "python3 -m pip install XXX" works.

Reply:
The python2 version uses pip, but python3 uses pip3.